extensions/amp-form/0.1/amp-form.js

```diff
@@ -99,6 +150,11 @@ export class AmpForm {
   /** @private */
   installSubmitHandler_() {
     this.form_.addEventListener('submit', e => this.handleSubmit_(e), true);
+    const inputs = this.form_.querySelectorAll('input,select,textarea');
```

Comment:
My [suggestion](https://github.com/ampproject/amphtml/pull/3823#discussion_r70005303) to put event handlers on the form element and let it propagate from the input up would solve that.